modules/swagger-codegen/src/main/java/io/swagger/codegen/CodegenConstants.java

```diff
@@ -82,4 +82,9 @@
 
     public static enum MODEL_PROPERTY_NAMING_TYPE {camelCase, PascalCase, snake_case, original}
 
+    public static final String MODEL_NAME_PREFIX = "modelNamePrefix";
+    public static final String MODEL_NAME_PREFIX_DESC = "Prefix that will be prepended to all model names. Default is the empty string.";
```

Comment:
I believe you also aware that another way to add the default value is `.defaultValue` in CliOption ([example](https://github.com/swagger-api/swagger-codegen/blob/d05596dad42731356ecc187b7ffec915cd31c524/modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/PerlClientCodegen.java#L81))

In this case, I'm fine with both approaches as the default is an empty string.